portafly/src/i18n/locales/en/shared.yml

```diff
@@ -3,7 +3,7 @@ logo_alt_text: Red Hat Integration logo
 logo_alt_text_description: Alt attribute of the main logo
 buttons:
   submit: Submit
-  cancel: Cancel
+  cancel: Cancel #-----duplicated string
```

Comment:
We can get rid of the duplicated strings in a following PR, once we finish the job in the `milestone_dev-accounts `branch